src/ImageSharp/Common/Helpers/UnitConverter.cs

```diff
@@ -30,6 +30,11 @@ internal static class UnitConverter
         /// </summary>
         private const double InchesInMeter = 1 / 0.0254D;
 
+        /// <summary>
+        /// The default resolution unit value.
+        /// </summary>
+        private const PixelResolutionUnit DefaultResolutionUnit = PixelResolutionUnit.PixelsPerInch;
```

Comment:
This is a breaking change and I'd like to understand why it was added. The default `PixelResolutionUnit` is not `PixelsPerInch` but `AspectRatio`

Reply:
This was changed by Ildar: f9570d37. I think he did it because the default ResolutionUnit of tiff is inch. See Tiff spec, page 18.
I need to check how it was before his change.

Reply:
We used aspect ratio in jpeg as default since the values we assigned match the spec. If it's not causing issues then let's keep the change.